gui/dig.lua

```diff
@@ -1760,13 +1945,16 @@ DigScreen = defclass(DigScreen, gui.ZScreen)
 DigScreen.ATTRS {
     focus_path = "dig",
     pass_pause = true,
+    initial_pause = false,
```

Comment:
Some tools might want to set this to `true`, but no tool should have a need to set it explicitly to `false`. Instead, the player should set the default pause state in `gui/control-panel`

Reply:
Oh I didn't know that functionality existed, cool